### PR TITLE
Fixes #36311 - allow configuring mod_status

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: manifests/config/apache.pp [hunk 1]
```diff
@@ -232,6 +236,10 @@
   include apache
   include apache::mod::headers
 
+  if $mod_status {
+    include apache::mod::status
+  }
```
[the remark this answers is not in the document]
Comment (evgeni) [reply manifests/config/apache.pp:241]: or is that ensured (lol) by puppet cleaning modules if they are not defined?

**Review by ekohl — CHANGES_REQUESTED** [review]
:-1: on including this in puppet-foreman. We shouldn't configure this via Foreman because it's the wrong layer. We also have content proxies with Apache and those wouldn't be monitored then.

Users may also want to tune the IP addresses. If we want this configurable, we can simply include it in our answers file as a top level module.

**Comment by evgeni** [comment]
Yeah, I can see the "but what about proxy" argument.

Wouldn't including it as a top level module in our installer expose it too much as an "API" without us being in charge of it?

**Comment by evgeni** [comment]
let's try https://github.com/theforeman/foreman-installer/pull/877 instead

**Pull request closed without merging**